packages/sources/stasis/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'Stasis'
+
+export const DEFAULT_BASE_URL = 'https://stasis.net/transparency/eurs-statement'
```

Comment:
```suggestion
export const DEFAULT_BASE_URL = 'https://stasis.net'
```

This should be changed to be an actual base rather than the full URL.

`/transparency/eurs-statement` would be the `url`.